doc/en/index.rst

```diff
@@ -17,7 +17,7 @@ The ``pytest`` framework makes it easy to write small, readable tests, and can
 scale to support complex functional testing for applications and libraries.
 
 
-**Pythons**: ``pytest`` requires: Python 3.6, 3.7, 3.8, 3.9, or PyPy3.
+**Pythons**: ``pytest`` requires: Python 3.7, 3.8, 3.9, or PyPy3.
```

Comment:
3.10 still seems to be missing from the list?

Reply:
Alternatively, how about:

```suggestion
``pytest`` requires: Python 3.7+ and PyPy3.
```

Reply:
Sorry, I forgot to fetch upstream changes and force pushed my local branch :man_facepalming: 
Fixed!